.github/workflows/avm.template.module.yml

```diff
@@ -19,7 +19,10 @@ on:
         type: string
         description: "Relative path to the module folder"
         required: true
-
+      customLocation:
```

Comment:
Since part of `workflowInput` I think it's not needed here

Reply:
fixed